planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -478,9 +481,6 @@ class PlanningSceneMonitor : private boost::noncopyable
   // Callback for a new planning scene msg
   void newPlanningSceneCallback(const moveit_msgs::PlanningSceneConstPtr &scene);
 
-  // Called to update the planning scene with a new message.
-  void newPlanningSceneMessage(const moveit_msgs::PlanningScene& scene);
```

Comment:
On Mon, Jun 13, 2016 at 03:42:43PM -0700, Dave Coleman wrote:
> just a guess, but wouldn't this break ABI too?

No, changing visibility private -> public does not break ABI.
As it turns out, in this case changing the ```void``` return
to ```bool``` also is ABI compliant as the function was private before.
I'll prepare the changes.